### PR TITLE
Fix beacon link...

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Beacon links..

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. on main, click a beacon link - or inspect it on the page. Notice two lines from the template missing.
2. apply patch and find link working as before - and with that build 38 potential!

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by actual clicking on local (DN)

### CODE REVIEW

File: scout/server/links.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 from scout.utils.convert import amino_acid_residue_change_3_to_1
 
 SHALLOW_REFERENCE_STR_LOCI = ["ARX", "HOXA13"]
-BEACON_LINK_TEMPLATE = "https://beacon-network.org/#/search?pos={this[position]}&"
-"chrom={this[chromosome]}&allele={this[alternative]}&"
-"ref={this[reference]}&rs={build}"
+BEACON_LINK_TEMPLATE = (
```
Comment (northwestwitch) [inline scout/server/links.py:7]: Looks much better already!

**Review by northwestwitch — APPROVED** [review]
Looks good, we have to test it anyway

File: scout/server/links.py [hunk 2]
```diff
@@ -609,6 +609,8 @@ def beacon_link(variant_obj: dict, build: int):
     """Compose link to Beacon Network."""
     url_template = BEACON_LINK_TEMPLATE
 
+    build = "GRCh38" if build == 38 else "GRCh37"
```
Comment (northwestwitch) [inline scout/server/links.py:612]: 👍🏻

File: tests/server/test_links.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 BUILD_37 = 37
 BUILD_38 = 38
 
- = "GRCh37"
+BEACON_BUILD_37 = "GRCh37"
```
Comment (northwestwitch) [inline tests/server/test_links.py:20]: No, now the other tests of this page will fail!

Comment (northwestwitch) [reply tests/server/test_links.py:20]: ah no, you added a new constant

Comment (dnil) [reply tests/server/test_links.py:20]: I dont think so - it would if I changed `BUILD` - but let's see. This is an exciting show is it not!

Comment (northwestwitch) [reply tests/server/test_links.py:20]: I think it's lunchtime

File: tests/server/test_links.py [hunk 4]
```diff
@@ -17,14 +17,17 @@
...
 BUILD_38 = 38
 
+BEACON_BUILD_37 = "GRCh37"
+BEACON_BUILD_38 = "GRCh38"
+
 
 def test_beacon_link_37(variant_obj):
     """Test building a beacon link for a variant in build 37"""
     # GIVEN a variant in genome build 37
     build = BUILD_37
```
Comment (northwestwitch) [inline tests/server/test_links.py:27]: This one can be removed now

File: tests/server/test_links.py [hunk 5]
```diff
@@ -36,7 +39,7 @@ def test_beacon_link_38(variant_obj):
     build = BUILD_38
```
Comment (northwestwitch) [inline tests/server/test_links.py:39]: And this

File: tests/server/test_links.py (hunk 4 above)
Comment (northwestwitch) [reply tests/server/test_links.py:27]: Or not..

Comment (northwestwitch) [reply tests/server/test_links.py:27]: Never mind, it's used later in the code

**Pull request merged**